tests/vmi_multus_test.go

```diff
@@ -336,7 +342,9 @@ var _ = Describe("Multus", func() {
 				checkInterface(vmiTwo, "eth0", "localhost:~#")
 
 				By("ping between virtual machines")
-				pingVirtualMachine(vmiOne, "10.1.1.2", "localhost:~#")
+				err := pingVirtualMachine(vmiOne, "10.1.1.2", "localhost:~#")
+				Expect(err).ToNot(HaveOccurred(), "VMI %s should be reachable", vmiOne.Name)
```

Comment:
If I understand correctly, this check expects connectivity between two specific VMI/s. Therefore, the message is wrong (as the VMI may have connectivity, but not to the other one).